src/main/java/core/basesyntax/LocalFileReader.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.IllegalFormatFlagsException;
+import java.util.List;
+
+public class LocalFileReader {
+    private static final String COMMA_DELIMITER = ",";
+    private static final String[] FILE_HEADER = new String[]{"type", "fruit", "quantity", "date"};
```

Comment:
```suggestion
    private static final String[] FILE_HEADER = {"type", "fruit", "quantity", "date"};
```
 you can write declaration shorter

Reply:
done